noir-projects/noir-contracts/bootstrap.sh

```diff
@@ -108,6 +108,7 @@ function compile {
     ../../noir/.rebuild_patterns \
     ../../avm-transpiler/.rebuild_patterns \
     "^noir-projects/noir-contracts/contracts/$contract/" \
+    "^noir-projects/noir-protocol-circuits/crates" \
```

Comment:
```suggestion
    "^noir-projects/noir-protocol-circuits/crates/types" \
```

We only use the types crate outside of `noir-protocol-circuits`, right? Seems like we can be more granular.